packages/core/utils/src/dal/mikro-orm/base-entity.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  BeforeCreate,
+  Entity,
+  OnInit,
+  OptionalProps,
+  PrimaryKey,
+} from "@mikro-orm/core"
+import { generateEntityId } from "../../common"
+
+@Entity({ abstract: true })
+export class MikroOrmBaseEntity {
```

Comment:
Would it make sense to just name it `BaseEntity`? What's the purpose of mentioning `MikroORM` in the name?

Reply:
Just to avoid confusion if someone want to use a different orm in their module

Reply:
But this just for some cases right? In general cases most people would use what we provide. For those that use a different ORM they can import it with a different name.

Not a big issue though, I just think since this is something that we're providing developers with (not something used internally) we don't need to be too explicit with the name. Easier names are easier to remember 😄

Reply:
Yes I agree, my thinking was that people dont think they can use it with prisma, the same way we prefixed our other utils for mikro orm. If you think it is better without I can remove it, just confirm me 😃

Reply:
Not sure about other utils, but let's just name this one `BaseEntity` 👍🏻

Reply:
👍 '

Reply:
done